TreeMaker/python/doPhotons.py

```diff
@@ -6,7 +6,20 @@ def doPhotonVars(self,process):
     process.goodPhotons = PhotonIDisoProducer.clone(
         conversionCollection   = cms.untracked.InputTag("reducedEgamma","reducedConversions",self.tagname)
     )
-
+    from TreeMaker.Utils.EgammaPostRecoTools import setupEgammaPostRecoSeq 
+    era = cms.string("2018-UL")
```

Comment:
for this to actually work, you need to use the exact syntax from here:
https://github.com/TreeMaker/TreeMaker/blob/Run2_2017_V17/TreeMaker/python/makeTreeFromMiniAOD_cff.py#L475-L478

Reply:
Added it.